tests/math.spec.ts

```diff
@@ -2037,6 +2041,56 @@ describe('Math', () => {
       assert.equal(apy, Infinity)
     })
   })
+  describe('calculate user daily rewards tests', () => {
+    it('active position 20% of liquidity', async () => {
+      const current = jsonArrayToTicks(usdcUsdhPoolSnapshot.ticksCurrentSnapshot)
+      const tickLower = 4
+      const tickUpper = 5
+
+      const { singleTickLiquidity } = calculateTokensAndLiquidity(
+        current,
+        usdcUsdhPoolSnapshot.currentTickIndex
+      )
+
+      const paramsApy: UserDailyRewardsParams = {
+        ticksCurrentSnapshot: current,
+        currentTickIndex: usdcUsdhPoolSnapshot.currentTickIndex,
+        rewardInTokens: 10000,
+        userLiquidity: { v: singleTickLiquidity.divn(5) },
+        duration: 10,
+        lowerTickIndex: tickLower,
+        upperTickIndex: tickUpper
+      }
+
+      const apy = calculateUserDailyRewards(paramsApy)
+      //10000 tokens / 10 days * 20% of liquidity = 2000 tokens
```

Comment:
`200` in comment. This var should name `dailyRewards` or  something like that

Reply:
done